### ISSUE TITLE
Totally can't work…

### ISSUE THREAD

**Issue by Yubuxin** [charliecm/obsidian-tidy-footnotes#5]
I press the command but there's nothing happened.

obsidian version: 0.15.6

**Comment by charliecm** [comment]
@Yubuxin Can you post an example of a note you're using the plugin on?

**Comment by Yubuxin** [comment]
@charliecm  Thanks for reply !
I have this problem with all my notes… 
When I press the command or click the hotkey, the only thing happens is that two blank lines be added to the bottom of my note.

**Comment by Yubuxin** [comment]
Just like this @charliecm 
This is the screenshot before I clicking the hotkey:
![B7QOFUO{T@HN(7~)CP80C)6](https://user-images.githubusercontent.com/87011715/182335207-028ebb4b-5b1d-4f2c-8509-937904780e71.png)

This is the screenshot after I clicking the hotkey:
![@CPDWFAL02LXR`PM3(1D2JV](https://user-images.githubusercontent.com/87011715/182335456-ca16ea19-73d4-456c-8cd3-503de43beddd.png)

Just two more blank lines.

**Comment by charliecm** [comment]
@Yubuxin Can you send me your Markdown source? I can't tell from the screenshots how you're inputting the footnotes.

**Comment by claremacrae** [comment]
I wonder whether that screenshot is is Reading view?
Should the note be in Live Previous or Source mode for the replacement to work?

**Comment by charliecm** [comment]
It should work in both live preview and source mode. Are you encountering any issue @claremacrae?

**Comment by claremacrae** [comment]
> It should work in both live preview and source mode. Are you encountering any issue @claremacrae?

No, I was trying to give the OP suggestions on what their problem might be...
